src/EnergyPlus/WaterUse.cc

```diff
@@ -1092,117 +1099,142 @@ namespace WaterUse {
+                                              "Verify temperature setpoints and schedules.");
+                        } else {
+                            ShowRecurringWarningErrorAtEnd(
+                                state,
+                                format(
+                                    "\"{}\" - Target water temperature should be less than or equal to the hot water temperature error continues...",
+                                    this->Name),
+                                this->TargetHWTempErrIndex,
+                                TempDiff,
+                                TempDiff);
+                        }
                     }
                 }
             } else {
-                this->HotMassFlowRate = this->TotalMassFlowRate * (this->TargetTemp - this->ColdTemp) / (this->HotTemp - this->ColdTemp);
-            }
-
-            if (this->HotMassFlowRate < 0.0) {
-                // Target temp is colder than the cold water temp; don't allow colder
-                this->HotMassFlowRate = 0.0;
-                TempDiff = this->ColdTemp - this->TargetTemp;
-                if (!state.dataGlobal->WarmupFlag) {
-                    // print error for variables of target water temperature
-                    ++this->TargetCWTempErrorCount;
-                    if (this->TargetCWTempErrorCount < 2) {
-                        ShowWarningError(
-                            state,
-                            format("CalcEquipmentFlowRates: \"{}\" - Target water temperature is less than the cold water temperature ({:.2R} C)",
-                                   this->Name,
-                                   TempDiff));
-                        ShowContinueErrorTimeStamp(state, "");
-                        ShowContinueError(state, format("...target water temperature       = {:.2R} C", this->TargetTemp));
-                        ShowContinueError(state, format("...cold water temperature       = {:.2R} C", this->ColdTemp));
-                        ShowContinueError(state, "...Note: target water temperature should be greater than or equal to the cold water temperature");
-                        ShowContinueError(state,
-                                          "...Target water temperature should be greater than or equal to the cold water temperature. "
-                                          "Verify temperature setpoints and schedules.");
-                    } else {
-                        ShowRecurringWarningErrorAtEnd(
-                            state,
-                            format(
-                                "\"{}\" - Target water temperature should be greater than or equal to the cold water temperature error continues...",
-                                this->Name),
-                            this->TargetCWTempErrIndex,
-                            TempDiff,
-                            TempDiff);
+                // Check hot less than cold temp, else calculate hot flow.
```

Comment:
and finally the else. if the hot temp is lower than cold temp, post warning, otherwise calculate hot flow.

Reply:
OK